### PR TITLE
#169258570 Setup react Router

### PR DESCRIPTION
**Pull request by william000000**
#### What does this PR do?
Setup react-router 

#### Description of Task to be completed?
  - Add set up of react-router in `src/index.js`
  - Add `NotFound.js` file to be displayed when page searched not exists

#### How should this be manually tested?
- Clone this repository using the command: `https://github.com/andela/team-odd-bn-frontend.git`
- Checkout to this branch using the command: `git checkout ch-setup-react-router-169258570`
- Install all dependencies `npm install`
- Run the application using the command: `npm run dev`

#### What are the relevant pivotal tracker stories?
[#169258570](https://www.pivotaltracker.com/story/show/169258570)
#### Screenshots (if appropriate)
image

image

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,7 +1,33 @@
...
+            Home
+          </NavLink>
+        </li>
+      </ul>
+
+      <Switch>
+        <Route exact path="/" component={App} />
+        <Route component={Notfound} />
+      </Switch>
+    </div>
```
Comment (NiyoEric) [inline src/index.js]: I recommend you to use `index.js` file to render our application and `App.js` to import all components and configure routes.

``` js
// index.js
ReactDOM.render(<App />, document.getElementById('root'));

//App.js
const App = () => {
  return (
      <Router>
        <div className="container">
          <Switch>
            <Route path="/" exact component={componentA} />
            <Route path="/signin" exact component={componentB} />
            <Route path="/signup" exact component={componentC} />
          </Switch>
        </div>
      </Router>
  );
}

```

File: src/index.js [hunk 2]
```diff
@@ -1,7 +1,33 @@
...
+const routing = (
+  <Router>
+    <div>
+      <ul>
+        <li>
+          <NavLink exact activeClassName="active" to="/">
+            Home
+          </NavLink>
+        </li>
+      </ul>
```
Comment (NiyoEric) [inline src/index.js]: I think the markup style is not needed. All routes will render components.

**Review by NiyoEric — CHANGES_REQUESTED** [review]
Good job @william000000! I have left feedback below. Also, I request you to update PR Readme with screenshots.

File: src/App.js [hunk 3]
```diff
@@ -1,8 +1,28 @@
+/* eslint-disable react/jsx-filename-extension */
```
Comment (hezronkimutai) [inline src/App.js]: Hi @ willy, remove this eslint comment and solve the problem instead

**Review by hezronkimutai — CHANGES_REQUESTED** [review]
Nice work willy, just check on the comments I left.

File: src/components/user/signout/signout.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const Signout = () => {
+    console.log('hell');
```
Comment (hezronkimutai) [inline src/components/user/signout/signout.js]: Remove the console

**Review by NiyongaboEric — APPROVED** [review]

**Review by NiyoEric — APPROVED** [review]

**Review by hezronkimutai — APPROVED** [review]

**Review by shemaeric — APPROVED** [review]
Good work @william000000

**Pull request merged**